tests/components/lcn/test_climate.py

```diff
@@ -0,0 +1,289 @@
+"""Test for the LCN climate platform."""
+
+from unittest.mock import patch
+
+from pypck.inputs import ModStatusVar, Unknown
+from pypck.lcn_addr import LcnAddr
+from pypck.lcn_defs import Var, VarUnit, VarValue
+
+# pylint: disable=hass-component-root-import
+from homeassistant.components.climate import DOMAIN as DOMAIN_CLIMATE
+from homeassistant.components.climate.const import (
+    ATTR_CURRENT_TEMPERATURE,
+    ATTR_HVAC_MODE,
+    ATTR_HVAC_MODES,
+    ATTR_MAX_TEMP,
+    ATTR_MIN_TEMP,
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    SERVICE_SET_HVAC_MODE,
+    SERVICE_SET_TEMPERATURE,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.components.lcn.helpers import get_device_connection
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    ATTR_SUPPORTED_FEATURES,
+    ATTR_TEMPERATURE,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .conftest import MockModuleConnection
+
+
+async def test_setup_lcn_climate(hass: HomeAssistant, entry, lcn_connection) -> None:
```

Comment:
entry is unused

I think this one would also be covered with the snapshto test